tests/components/roborock/test_config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Test Roborock config flow."""
+from unittest.mock import patch
+
+from roborock.exceptions import RoborockException
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.roborock.const import CONF_ENTRY_CODE, DOMAIN
+from homeassistant.core import HomeAssistant
+
+from .mock_data import MOCK_CONFIG, USER_EMAIL
+
+
+async def config_flow_helper(
```

Comment:
Test helpers like this make tests harder to read and are discouraged. Instead, use either test fixtures, parameterized tests, or just duplicate code between tests.